tests/cephfs/clients/mds_scrub_after_failover.py

```diff
@@ -0,0 +1,144 @@
+        }
+        # fill up to 60% of the cluster
+        fs_io(client=clients[0], fs_config=cephfs, fs_util=fs_util)
+        # Prepare the clients with different mounting options
+        # set max mds 2
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs max_mds 2")
+        # set standby count to 2
+        client1.exec_command(sudo=True, cmd="ceph fs set cephfs standby_count_wanted 2")
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        kernel_mounting_dir_2 = f"/mnt/cephfs_kernel{mounting_dir}_2/"
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        fuse_mounting_dir_2 = f"/mnt/cephfs_fuse{mounting_dir}_2/"
+        mon_node_ips = fs_util.get_mon_node_ips()
+
+        default_fs = "cephfs"
+
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mounting_dir_1,
+            extra_params=f" --client_fs {default_fs}",
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mounting_dir_2,
+            extra_params=f" --client_fs {default_fs}",
+        )
+
+        fs_util.kernel_mount(
+            [client1],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+            extra_params=f",fs={default_fs}",
+        )
+        fs_util.kernel_mount(
+            [client1],
+            kernel_mounting_dir_2,
+            ",".join(mon_node_ips),
+            extra_params=f",fs={default_fs}",
+        )
+        # getting all the MDS info to fail one of the MDS
+        result, rc = clients[0].exec_command(
+            sudo=True, cmd=f"ceph fs status {default_fs} --format json-pretty"
+        )
+        mds_name_list = []
+        result_json = json.loads(result)
+        active_mds_before = []
+        for elem in result_json["mdsmap"]:
+            if elem["state"] == "active":
+                mds_name_list.append(elem["rank"])
+                active_mds_before.append(elem["name"])
+        log.info(f"Active mds before mds fail {active_mds_before}")
+        time.sleep(10)
+
+        with parallel() as p:
+            p.spawn(fs_util.run_ios, client1, fuse_mounting_dir_1)
+            p.spawn(fs_util.run_ios, client1, fuse_mounting_dir_2)
+            p.spawn(fs_util.run_ios, client1, kernel_mounting_dir_1)
+            p.spawn(fs_util.run_ios, client1, kernel_mounting_dir_2)
+        # kill active MDS
+        client1.exec_command(sudo=True, cmd=f"ceph mds fail {active_mds_before[-1]}")
+        time.sleep(10)
+        rc, ec = client1.exec_command(
+            sudo=True,
+            cmd="ceph tell mds.cephfs:0 scrub start / recursive -f json-pretty",
+        )
+        result = json.loads(rc)
+        log.info(result)
+        if result["return_code"] != 0:
+            log.error("Error while scrubbing")
+            return 1
+
```

Comment:
Can you please add a check for ceph status if there is any "damaged metadata" error logged. Good to log ceph health as well for debugging purposes.

Reply:
Trying to add "damaged metadata" from  "ceph -s", do you know where it has the "damaged metadata"..?

Reply:
We have to check if the "damaged metadata" is not logged in "ceph -s" output.